### ISSUE TITLE
debug logging too verbose

### ISSUE THREAD

**Issue by tethlah** [bbindreiter/node-red-contrib-alexa-remote2-applestrudel#187]
It seems like enabling debug logging is outputting what I should be seeing at trace level.  I'm trying to troubleshoot an issue with something else, but the debug output is being flooded by the responses from alexa.  There's no deference between debug and trace with the alexa output.  The fully verbose outputs should be in trace and not debug.

**Comment by tethlah** [comment]
So the issue I've been trying to track down is that my node red instance keeps randomly restarting a few times per day over the last couple weeks.  Since the log was flooded by the Alexa node, I decided to disable it to clear up my debug log so I could troubleshoot what's happening just before restart.  It hasn't restarted since I disabled the Alexa Config node.  So it looks like something with this node is what's causing my constant reboots.  It's pretty hard debug what's going on though because the logs are flooded with the Alexa responses from the requests in the node.
